### ISSUE TITLE
override bug with data.frame

### ISSUE THREAD

**Issue by jordansread** [USGS-R/gsplot#322]
Our rule to still fully support the masked `graphics` functions is failing in the data.frame case (probably others?)

``` r
# this works as expected
graphics::plot(1,3)
graphics::points(data.frame(1,2), col='red')

# this should have the same behavior, but errors instead:
plot(1,3)
points(data.frame(1,2), col='red')
 Error in xy.coords(x, y) : argument "x" is missing, with no default 
```

**Comment by lindsayplatt** [comment]
y labels not showing anymore

**Comment by jordansread** [comment]
@lindsaycarr can you provide more detail for this?

**Comment by lindsayplatt** [comment]
see my comment in #411 on Nov 7

**Comment by jordansread** [comment]
thanks

**Comment by jordansread** [comment]
Confused on this one. Working from my `data_frame` branch:

``` r
gs <- gsplot() %>% points(1,2)
setdiff(gs$side.2$axis, gs$side.1$axis)
[[1]]
[1] 2
# so not different other than the side arg in axis
```

And the other args are the same other than limits (between sides 2 and 3) 

So I am thinking this issue must be in `print` or `draw_axis`. 

Same thing happening here https://github.com/USGS-R/gsplot/issues/414#issuecomment-261083821 @lindsaycarr ?

**Comment by lindsayplatt** [comment]
@jread-usgs - @jiwalker-usgs and I were coming across this same axis labels disappearing issue yesterday when working in print. See my [last comment](https://github.com/USGS-R/gsplot/issues/414#issuecomment-261083821) on #414. We were stumped.

**Comment by lindsayplatt** [comment]
@jread-usgs I've been debugging this for awhile, and replacing the if-else section with a method `params <- append_params(object, params)` is what causes the labels to disappear. I haven't been able to get much farther past identifying what causes the issue.

**Comment by lindsayplatt** [comment]
That disappearing labels thing seemed unrelated to this now that we fixed it for #414. Maybe we should retry the fix that originally existed for this? Currently trying to figure out how to pull it down and test.

**Comment by lindsayplatt** [comment]
If this is just for allowing regular graphics plot stuff to work, this branch fixes it (https://github.com/lindsaycarr/gsplot/commit/cae685c5ff568a076d07b175a5103aecaf52dd29). It does not allow gsplot to use data frames.

```r
gs <- gsplot() %>% 
    points(data.frame(1,2))
Error in is.finite(side.vals) : 
  default method not implemented for type 'list'
```
